mocca-client/src/test/java/com/paypal/mocca/client/MoccaSerializerTest.java

```diff
@@ -141,6 +143,119 @@ public void complexResponseCustomSelectionSetLessFieldsTest() throws IOException
                 selectionSet, "{\n  \"query\" : \"query{getABeer(sampleRequest: {bar: \\\"bar\\\", foo: \\\"foo\\\"}) {booleanVar complexField stringVar}}\"\n}");
     }
 
+    @Test
+    public void complexWithStringListRequestTest() throws IOException {
+        SuperComplexSampleType.SuperComplexField superComplexField =
+                new SuperComplexSampleType.SuperComplexField(77, "sevenseven", false,
+                        Arrays.asList(new String[] {"cat", "dog", "monkey"}), null, null);
+        List<SuperComplexSampleType.SuperComplexField> listOfComplexSampleTypes = new ArrayList<>();
+        SuperComplexSampleType superComplexSampleType = new SuperComplexSampleType(7, "seven", true,
+                superComplexField, listOfComplexSampleTypes, null);
+
+        List<MoccaSerializer.Variable> variables = Collections.singletonList(
+                new MoccaSerializer.Variable( superComplexSampleType, SuperComplexSampleType.class, newVar("sampleRequest")));
+
+        requestTest(variables, SampleResponseDTO.class, "getOneComplexSample", MoccaUtils.OperationType.Mutation,
+                null,  "{\n  \"query\" : \"mutation{getOneComplexSample(sampleRequest: {booleanVar: true, complexField: {innerBooleanVar: false, innerIntVar: 77, innerStringListVar: [\\\"cat\\\", \\\"dog\\\", \\\"monkey\\\"], innerStringVar: \\\"sevenseven\\\"}, complexListVar: [], intVar: 7, stringVar: \\\"seven\\\"}) {bar foo}}\"\n}");
```

Comment:
it should be `query{getOneComplexSample` instead of `mutation{getOneComplexSample`

Please do the same change in the other tests you added.

Reply:
ok